components/google_sheets/google_sheets.app.js

```diff
@@ -127,11 +127,12 @@ module.exports = {
       }
```

Comment:
Convert `google_sheets.app.js` and `sources/common.js` from [CommonJS to ESM](https://pipedream.com/docs/components/guidelines/#components-should-be-es-modules)?

Reply:
When I tried updating those files to ESM, I got the error
```
error         | component create api call responded with status: 500, body: {"error":"unexpected error, please contact support@pipedream.com"}
```
Also mentioned in [this PR comment](https://github.com/PipedreamHQ/pipedream/pull/1852#discussion_r739519606)

Reply:
I not longer get the error mentioned above when using [`pd dev`](https://pipedream.com/docs/cli/reference/#pd-dev), [`pd publish`](https://pipedream.com/docs/cli/reference/#pd-publish), or [`pd deploy`](https://pipedream.com/docs/cli/reference/#pd-deploy) on Google Sheets components after converting the app and common files to ESM.